src/rest/rest-api.php

```diff
@@ -128,8 +128,6 @@ public static function get_block_content( $params ) {
 		$post_id = $params['id'];
 		$post    = get_post( $post_id );
 
-		Analytics::record_usage();
```

Comment:
?

Reply:
Due to the way we were recording usage, if someone decided to call the `ContentParser` directly instead of going via the rest api the usage would not be counted. By moving the usage tracker into the `ContentParser` it avoids that problem. Plus it works for GraphQL that way as well, since that calls that code as well.